changelog.d/344.fixed.md

```diff
@@ -0,0 +1 @@
+Fix using wrong value to check if event is correct subclass
```

Comment:
This is not specific enough, from just reading this I have no idea which part of Zino has been changed

```suggestion
When matching an event to a planned maintenance check that event is of the correct subclass
```